osp-workshop-2024.sh

```diff
@@ -60,14 +66,38 @@ there is proper $kubeconfig_file provided and that at least one OpenStack DataPl
 EOF
             exit 1
         fi
-        # Now lets remove from the inventory file lines about ssh key which
-        # is mounted in the ansible runner pod but we want to use key provided
-        # with the -p parameter instead
-        sed -i '/ansible_ssh_private_key_file: /d' $inventory_file
+       # Now lets build very simple inventory file with just required options
+       python3 prepare_inventory.py $tmp_edpm_inventory_file $inventory_file
+    fi
+}
+
+function configure_ssh_jump_host() {
+    grep -q $jump_host $SSH_CONFIG
+    if [ $? -eq 0 ]; then
```

Comment:
Nit: if you just want to check if the jump host was already configured you can do it in one line
`
if grep -q $jump_host $SSH_CONFIG; then
`

Reply:
Maybe searching for `"ProxyJump $jump_host"` is more robust in case the config already contains the jump host for any reason?

Reply:
Done